src/openMVG/multiview/solver_resection_p4pf.hpp

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
I already made many review there but I think they have been ignored.
Please not that the header from opengv is not required here since it's some new code